app/src/main/java/com/example/android/weatherapp/models/CurrentWeatherModel.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.android.weatherapp.models
+
+import android.service.controls.templates.TemperatureControlTemplate
```

Comment:
What is this import supposed to do?

Reply:
It is an unused import added while adding **temperature** variable. I have removed it.